.travis.yml

```diff
@@ -1,5 +1,5 @@
-sudo: false
```

Comment:
You can't remove this on existing repos. Only on new ones. That's how Travis deal with the transition.